tests/providers/test_smtp.py

```diff
@@ -51,6 +51,15 @@ def test_multiple_to(self, provider):
         rsp = provider.notify(to=to, message='foo')
         assert rsp.data['to'] == ','.join(to)
 
+    def test_attachment(self, provider, tmpdir):
+        dir_ = tmpdir.mkdir('sub')
+        file_1 = dir_.join('foo.txt')
+        file_2 = dir._join('bar.txt')
+        file_3 = dir._join('baz.txt')
+        attachments = [file_1, file_2, file_3]
```

Comment:
you need to add `strpath` to the file, that's how `LocalPath` works...

Reply:
I was obviously in a hurry.  And I haven't been using LocalPath since pathlib was introduced.